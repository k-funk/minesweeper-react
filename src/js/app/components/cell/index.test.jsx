import React from 'react';
import { shallow } from 'enzyme';
import shallowToJson from 'enzyme-to-json';

import Cell from './index';


describe('outputs the expected tree when', () => {
  let wrapper;

  test('(default)', () => {
    wrapper = shallow((
      <Cell cellData={{}} onClick={() => {}} />
    ));
  });

  afterEach(() => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
