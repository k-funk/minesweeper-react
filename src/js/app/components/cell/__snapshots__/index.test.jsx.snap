// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`outputs the expected tree when (default 1`] = `
<div
  className=""
>
  Hello World!
  <Loader
    className=""
    loading={true}
    small={false}
  />
</div>
`;
